src/consumer/component/consumer-component.js

```diff
@@ -641,14 +634,20 @@ export default class Component {
         specs = path.join(bitDir, bitJson.getSpecBasename());
       }
     }
-    const distPath = path.join(process.cwd(), bitJson.distEntry);
-    const entry = fs.existsSync(distPath) ? distPath : process.cwd();
+    const enrtyPath = path.join(process.cwd(), bitJson.distEntry);
```

Comment:
Didn't we already have this logic in another place?  (during the build)
If yes export it to a function